### PR TITLE
Regex matching instead of traditional == matching :)

### PR DESCRIPTION
**Pull request by baalajimaestro**
Tested filters, test blacklist(must work 99%) just incase, before merge :3

### CODE REVIEW

**Review by skittles9823 — CHANGES_REQUESTED** [review]
Do these the lemme know if you tested it.

File: tg_bot/modules/blacklist.py [hunk 1]
```diff
@@ -112,24 +112,22 @@ def unblacklist(bot: Bot, update: Update):
 def del_blacklist(bot: Bot, update: Update):
     chat = update.effective_chat  # type: Optional[Chat]
     message = update.effective_message  # type: Optional[Message]
-    to_match = extract_text(message)
-    if not to_match:
+    match_string = extract_text(message)
+    if not match_string:
         return
-
+    to_match=match_string.split(" ")
```
Comment (skittles9823) [inline tg_bot/modules/blacklist.py]: No spaces between =

File: tg_bot/modules/blacklist.py [hunk 2]
```diff
@@ -112,24 +112,22 @@ def unblacklist(bot: Bot, update: Update):
...
     chat_filters = sql.get_chat_blacklist(chat.id)
     for trigger in chat_filters:
-        pattern = r"( |^|[^\w])" + re.escape(trigger) + r"( |$|[^\w])"
-        if re.search(pattern, to_match, flags=re.IGNORECASE):
-            try:
+        for word in to_match:
+            if re.fullmatch(keyword,word,flags=re.IGNORECASE):
```
Comment (skittles9823) [inline tg_bot/modules/blacklist.py]: Separate args with comma then space please.

File: tg_bot/modules/cust_filters.py [hunk 3]
```diff
@@ -145,27 +145,33 @@ def stop_filter(bot: Bot, update: Update):
...
+    if not match_string:
         return
-
+    to_match = match_string.split(" ")
     chat_filters = sql.get_chat_triggers(chat.id)
     for keyword in chat_filters:
-        pattern = r"( |^|[^\w])" + re.escape(keyword) + r"( |$|[^\w])"
-        if re.search(pattern, to_match, flags=re.IGNORECASE):
+       for word in to_match:
+         if re.fullmatch(keyword,word, flags=re.IGNORECASE):
```
Comment (skittles9823) [inline tg_bot/modules/cust_filters.py]: Separate args with comma then space please

**Comment by gotenksIN** [comment]
Peru

**Comment by yash-garg** [comment]
@songotenks69 K pro K

File: tg_bot/modules/blacklist.py (hunk 2 above)
Comment (baalajimaestro) [reply tg_bot/modules/blacklist.py]: Ok. Formatting ;-;

**Comment by baalajimaestro** [comment]
I tested filters, will test blacklist also

**Comment by skittles9823** [comment]
@baalajimaestro Naisu

**Comment by SphericalKat** [comment]
@skittles9823 What a wao

File: tg_bot/modules/blacklist.py (hunk 1 above)
Comment (baalajimaestro) [reply tg_bot/modules/blacklist.py]: Oooof. You know I am bad at formatting ;-;

**Comment by baalajimaestro** [comment]
Ok. Return back to shitpostclub @skittles9823 @ATechnoHazard 😂

**Comment by nyancrimew** [comment]
@baalajimaestro Wew

**Comment by nitanmarcel** [comment]
@deletescape .info

**Comment by baalajimaestro** [comment]
Pushed, fixed formatting, tested to work :)
Can be merged. @skittles9823

**Comment by yash-garg** [comment]
@baalajimaestro OwO

**Review by yash-garg — APPROVED** [review]

**Comment by rupansh** [comment]
.casefold does the same. Does it not?

**Comment by skittles9823** [comment]
Whao

**Pull request merged**